study-group/Google-Toast-Implementations.md

```diff
@@ -0,0 +1,101 @@
+# Google Toast Implementations
```

Comment:
Although Google implementations is a natural place to start (as a Google employee), for a public standards process, it's not where we want to end up; we shouldn't favor any single vendor in our end product. So, we should move this file away from being Google-specific.

It might also help to have TODO headings for the others in the same file.

Reply:
Thanks for the feedback, that completely makes sense, I'll overhaul it that way now.